anomalib/models/components/feature_extractors/timm.py

```diff
@@ -23,9 +23,9 @@ class FeatureExtractor(nn.Module):
 
     Example:
         >>> import torch
-        >>> from anomalib.core.model.feature_extractor import FeatureExtractor
+        >>> from anomalib.models.components.feature_extractors import TimmFeatureExtractor
 
-        >>> model = FeatureExtractor(model="resnet18", layers=['layer1', 'layer2', 'layer3'])
+        >>> model = TimmFeatureExtractor(model="resnet18", layers=['layer1', 'layer2', 'layer3'])
```

Comment:
I didn't duly read this PR but i saw this and wanted just to point a general suggestion.

I think that wherever this type of documentation examples include imports and/or anomalib classes the docstring should just point to one of those `.ipynb` tutorials/examples (instead of having a functional example in the docstring).

I think this would be less painful to update as the package evolves quite fast -- it occured to me more than twice that the example I got was not actually function.

Reply:
Maybe it would be a good idea to have both. Personally, I like to see the usage in hints and it also makes autodoc more descriptive. While ipynb files are good for getting started, I feel they introduce an extra overhead for maintenance. If there are broken examples then that's something to keep in mind on our part and future PR reviews. Examples make the method more readable so I am in favour of keeping these.